app/Http/Controllers/CurationExportController.php

```diff
@@ -15,6 +15,8 @@ public function __construct(
 
     public function getCsv(Request $request)
     {
+        $this->exporter->getCsv($request->all());
+        return view('main-test', []);
```

Comment:
not sure what is going on here with two different return statements...

Reply:
Thanks for catching this.  Debugging code that should have been removed.